src/client.js

```diff
@@ -325,6 +325,8 @@ class MistralClient {
         }
       }
     }
+
+    return response;
```

Comment:
Both unit tests and **chat_with_streaming** example pass with and without this one.